src/App.jsx

```diff
@@ -1,107 +1,31 @@
 import './App.scss';
 
-// import postsFromServer from './api/posts.json';
-// import commentsFromServer from './api/comments.json';
-// import usersFromServer from './api/users.json';
+import postsFromServer from './api/posts.json';
+import commentsFromServer from './api/comments.json';
+import usersFromServer from './api/users.json';
+import { PostList } from './components/PostList';
+
+function getUserById(userId) {
+  return usersFromServer.find(user => user.id === userId)
+    || null;
+}
+
+function getCommentById(postId) {
+  return commentsFromServer.filter(comment => comment.postId === postId)
+    || null;
```

Comment:
```suggestion
  return commentsFromServer.filter(comment => comment.postId === postId);
```
redundant as filter returns array. even empty array will be given to true and code || null  never will be executed.